arroyo/processing/processor.py

```diff
@@ -79,21 +79,24 @@ def _create_strategy() -> None:
 
         def on_partitions_assigned(partitions: Mapping[Partition, int]) -> None:
```

Comment:
The signature of this method says that partitions cannot be optional. If what you mention is that partitions could be `None` then can you update the signatures of these functions

Reply:
I don't think it would be `None`, just an empty dictionary.

Reply:
Yeah, it's an empty dictionary